gradle/versions.gradle

```diff
@@ -15,27 +15,35 @@
 
 dependencyManagement {
   dependencies {
-    dependency 'com.fasterxml.jackson.core:jackson-databind:2.12.4'
-    dependency 'com.fasterxml.jackson.datatype:jackson-datatype-jdk8:2.12.4'
+    dependencySet(group: 'com.fasterxml.jackson.core', version: '2.13.1') {
+      entry 'jackson-databind'
+      entry 'jackson-datatype-jdk8'
+    }
+
+    dependency 'com.github.ben-manes.caffeine:caffeine:3.0.5'
 
-    dependency 'com.github.tomakehurst:wiremock-jre8-standalone:2.29.1'
-    dependency 'com.github.tomakehurst:wiremock-jre8:2.29.1'
+    dependencySet(group: 'com.github.tomakehurst', version: '2.29.1') {
+      entry'wiremock-jre8-standalone'
+      entry'wiremock-jre8'
+    }
 
-    dependency 'com.google.auto.service:auto-service:1.0'
+    dependency 'com.google.auto.service:auto-service:1.0.1'
 
-    dependency 'com.google.dagger:dagger-compiler:2.30.1'
-    dependency 'com.google.dagger:dagger:2.30.1'
+    dependencySet(group: 'com.google.dagger', version: '2.40.5') {
+      entry'dagger-compiler'
+      entry'dagger'
+    }
 
-    dependencySet(group: 'com.google.errorprone', version: '2.8.0') {
+    dependencySet(group: 'com.google.errorprone', version: '2.10.0') {
       entry 'error_prone_annotation'
       entry 'error_prone_check_api'
       entry 'error_prone_core'
       entry 'error_prone_test_helpers'
     }
 
-    dependency 'com.google.guava:guava:30.1.1-jre'
-    dependency 'com.github.ben-manes.caffeine:caffeine:3.0.4'
```

Comment:
we delete because it is no longer used ?

Reply:
Moved to an alpha sort by group.  These were out of order.